doc/guide/pd/polydispersity.rst

```diff
@@ -86,6 +86,82 @@ Distribution but this may not be suitable. See** `Suggested Applications`_ **bel
            proportional size distributions and orientational distributions used in
            SasView models.
 
+Calculation of I(q)
+^^^^^^^^^^^^^^^^^^^
+
+Let $w(r)$ be the *relative number* of particles of size $r$, **not the volume
+fraction of particles**. $w(r)$ scales with the number density, $n(r)$.
+
+The *volume fraction*, $φ$, is the integrated volume of all particles, $V_p$, divided
+by total volume, $V_t$
+
+.. math:: 
+     :label: eq1
+
+     φ = \frac{V_p}{V_t}
+
+where $V_p$ is the number of particles, $N$, multiplied by the average particle volume
+$\langle V(r) \rangle$
+
+.. math::
+     :label: eq2
+
+     Vp = N \langle V(r) \rangle
+
+The *number density* of particles, $n$, is the total number of particles divided by
+the total volume
+
+.. math::
+     :label: eq3
+
+     n = \frac{N}{V_t}
+
+Since $w(r)$ is a distribution on the number of particles that (ideally) sums to one,
+the number of particles of size $r$, $n(r)$, scales with $w(r)$ as
+
+.. math::
+     :label: eq4
+
+     n(r) = \frac{w(r)}{∫w(r)dr} \cdot \frac{N}{V_t}
```

Comment:
The unicode integral does not act like latex `\int` with regard to automatic sizing based on the contents. Similarly for `\sum` and `\prod`. :-(

Also, I think latex produces math greek, which is in a different part of the table from modern Greek and so my render differently depending on font. That means ugly `\phi` in the markup to get things consistent. Also latin symbols. See code block U+1D400 to U+1D7FF.

I guess you could produce a doc with `unicode φ  latex unicode $φ$ latex $\phi$` to see if they render to different symbols. You will need to look at the entity value rather than the character shape if they look the same. Different code points can share characters depending on the font.

Reply:
Ok. Have now utilised \phi and \int.